luda-adventure/src/app/game/types/game_object/types/positioner/heading.rs

```diff
@@ -0,0 +1,24 @@
+use crate::app::game::{TileExt, Velocity};
+use namui::prelude::*;
+
+pub enum Heading {
+    Left,
+    Right,
+}
+
+pub fn get_heading_from_velocity(velocity: Velocity) -> Option<Heading> {
+    let unit_delta_x = velocity.x * 1.ms();
+    let unit_delta_y = velocity.y * 1.ms();
+
+    if unit_delta_x == 0.tile() {
+        return None;
+    }
+    let tangent = unit_delta_y / unit_delta_x;
+    if tangent.abs() > 8.0 {
```

Comment:
![image](https://user-images.githubusercontent.com/38313680/193456162-37af8148-2c86-4912-91ac-a4a058d9d495.png)
*abs(tan) > 8

Reply:
What's the purpose of this gray dead-zone?

Reply:
There is no important reason. 
When I was playing Among Us with a joystick, it bothered me to see it flip from side to side often while moving up.